third_party/docfx/templates/devsite/partials/uref/namespace.examples.tmpl.partial

```diff
@@ -10,7 +10,7 @@
 {{#codeexamples}}
 {{#name}}
 {{^name.0}}
-<h6 class="notranslate">{{name}}</h6>
+<div class="notranslate"><strong>{{name}}</strong></div>
```

Comment:
Semantically, I think this should be a header?

Reply:
It doesn't quite align with the rest of the TOC on right nav that it shows up as a header. I think it's fine to leave it out, currently it only affects the goldens for Go however.